crates/fluvio-version-manager/tests/install-fvm.sh

```diff
@@ -0,0 +1,448 @@
+#!/bin/bash
+# shellcheck shell=bash
+
+# test install.sh command for developer tryout
+
+set -e
+set -o pipefail
+set -u
+
+readonly FVM_INSTALL_DIR=${FVM_INSTALL_DIR-"$HOME/.fvm"}
+readonly FLUVIO_INSTALL_DIR=${FLUVIO_INSTALL_DIR-"$HOME/.fluvio"}
+readonly FLUVIO_ARCH=${FLUVIO_ARCH-}
+readonly FLUVIO_VERSION=${FLUVIO_VERSION-}
+readonly VERSION=${VERSION-}
+
+_fluvio_version="${FLUVIO_VERSION:-${VERSION:-}}"
+
+# install fvm
+main() {
+    need_cmd curl
+
+	# Detect architecture and ensure it's supported
+	get_architecture || return 1
+	local _arch="$RETVAL"
+	assert_nz "$_arch"
+
+    if [ -z "${FLUVIO_VERSION}" ] && [ "${VERSION}" != "" ]; then
+        echo "Warning: VERSION is deprecated in favor of FLUVIO_VERSION"
+    fi
+
+    # Some architectures may be folded into a single 'target' distribution
+    # e.g. x86_64-unknown-linux-musl and x86_64-unknown-linux-gnu both download
+    # the musl target release. The _target here is used in the URL to download
+    _target=$(normalize_target ${_arch})
+
+    echo "☁️  Downloading fluvio version manager, fvm"
+    echo "   target arch ${_target}"
+
+    _dir="$(mktemp -d 2>/dev/null || ensure mktemp -d -t fluvio-install)"
+    _temp_file="${_dir}/fvm.zip"
+    # todo: switch to hub/or fluvio packages download w/ checksum verification
+    _url="https://github.com/infinyon/fluvio/releases/download/dev/fvm-${_target}.zip"
+
+    # ".zip" will be removed in switch to hub/fluviopkgs
+    downloader "${_url}" "${_temp_file}.zip"
+    _status=$?
+    if [ $_status -ne 0 ]; then
+        err "❌ Failed to download fvm!"
+        err "    Error downloading from ${_url}"
+        abort_prompt_issue
+    fi
+
+    # ".zip" will be removed in switch to hub/fluviopkgs
+    unzip -qq -d ${_dir} ${_temp_file}
+    chmod +x ${_dir}/fvm
+
+	echo "⬇️ Installing fvm"
+    ${_dir}/fvm self install
+
+    #
+    # check if .fluvio exists, recommend fvm install
+    echo "fluvio install dir $FLUVIO_INSTALL_DIR"
+    if [ -d "$FLUVIO_INSTALL_DIR" ]; then
+        echo "A version fo fluvio is already installed, please run 'fvm install' or 'fvm switch' to change versions"
```

Comment:
nit: by introducing https://github.com/infinyon/fluvio/pull/3604 `fvm install` will be enough to install Fluvio. At this point `fvm switch` will not have any local versions to switch to.